dist/css/cookie-compliance-for-wordpress.css

```diff
@@ -27,8 +27,8 @@
   left: 0;
   max-width: 100%;
   display: block;
-  border-bottom: 2px solid var(--xDark, black);
-  background-color: var(--light);
+  border-bottom: 2px solid black;
+  background-color: #f3f2f1;
   padding: 30px 10px;
```

Comment:
A note on the above, I've removed variables on these specific css properties because they make a big difference when viewed in IE11. The rest of the variables are more cosmetic so I've left them for now. I would prefer to keep using variables if possible.